TypeCobol.Test/Parser/Programs/Cobol85/SetToFalse.Mix.txt

```diff
@@ -0,0 +1,76 @@
+       IDENTIFICATION DIVISION.
+       PROGRAM-ID. SetToFalse.
+       ENVIRONMENT DIVISION.
+       CONFIGURATION SECTION.
+       SOURCE-COMPUTER. IBM-370
+      *                WITH DEBUGGING MODE
+                      .
+       SPECIAL-NAMES.
+           DECIMAL-POINT IS COMMA
+           .
+       DATA DIVISION.
+       WORKING-STORAGE SECTION.
+
+       01 Var1 pic X.
+           88 ok1 value 'A' false 'B'.
+           88 ok2 value 'A' false 'C'.
+           88 ok3 value 'A1' 'A2' false IS 'D'.
+           88 ok4 value 'A' when set to false is 'E'.
+           88 ok5 value 'A' set false 'F'.
+           88 ok6 value 'A' to false 'G'.
+           88 ok7 value 'A' when false 'H'.
+           88 ok8 value 'A' when to false 'I'.
+      * Invalid clause WHEN SET TO FALSE
+Line 24[37,39] <27, Error, Syntax> - Syntax error : missing FALSE at ''U''
+           88 ko1 value 'A' when to 'U'.
+Line 25[41,42] <27, Error, Syntax> - Syntax error : missing FALSE at 'is'
+           88 ko2 value 'A' when set to is 'V'.
+Line 26[33,35] <27, Error, Syntax> - Syntax error : mismatched input ''W'' expecting {FALSE, TO}
+           88 ko3 value 'A' set 'W'.
+Line 27[32,34] <27, Error, Syntax> - Syntax error : missing FALSE at ''X''
+           88 ko4 value 'A' to 'X'.
+Line 28[34,36] <27, Error, Syntax> - Syntax error : mismatched input ''Y'' expecting {SET, FALSE, TO}
+           88 ko5 value 'A' when 'Y'.
+Line 29[34,35] <27, Error, Syntax> - Syntax error : mismatched input 'is' expecting {SET, FALSE, TO}
+           88 ko6 value 'A' when is 'Z'.
+Line 30[49,49] <27, Error, Syntax> - Syntax error : mismatched input '.' expecting {alphanumeric literal, numeric literal, figurative constant, keyword}
+           88 ko7 value 'A' when set to false is.
+      * Missing clause WHEN SET TO FALSE
+           88 ko8 value 'A'.
```

Comment:
We don't expect an error in the declaration itself but SET this variable TO FALSE will fail because the clause WHEN SET TO FALSE is missing.
Comment changed: is it better now?

Reply:
yes